third_party/model_api/model_api.bzl

```diff
@@ -131,6 +131,7 @@ def _impl(repository_ctx):
         
     # Note we need to escape '{/}' by doubling them due to call to format
     if _is_windows(repository_ctx):
+        openvino_dir = openvino_dir.replace("/", "\\\\").replace("\\", "\\\\")
```

Comment:
Won't this double the amount of `\`? You add new ones in the first replace and then they get duplicated by the second replace. 
Isn't there some Python built-in to deal with it?